components/TokenManagePage/OwnerWarning.tsx

```diff
@@ -0,0 +1,27 @@
+"use client";
+import { Fragment } from "react";
+import { getAddress } from "viem";
+import { useTokenProvider } from "@/providers/TokenProvider";
+import { useUserProvider } from "@/providers/UserProvider";
+
+const OwnerWarning = () => {
+    const { connectedAddress } = useUserProvider();
+    const { owner } = useTokenProvider();
+
+    const isOwner = Boolean(
+        connectedAddress && owner &&
+        getAddress(connectedAddress) === getAddress(owner)
+    );
+
+    if (isOwner) {
+        return <Fragment />
+    }

```

Comment:
This could be simplified

```suggestion
    if (isOwner) return null;
```